### PR TITLE
Fix PhpStan level 5 errors

### PR DESCRIPTION
**Pull request by blankse**
Before:
`[OK] Baseline generated with 414 errors.`
After:
`[OK] Baseline generated with 62 errors.`

- [x] Rebase/Merge if splitted PRs are merged and in 10.x branch
- [x] Create more splitted PRs

### CODE REVIEW

**Comment by dvesh3** [comment]
@blankse we are thinking of increasing the Stan level to 5 on 10.4 release, which is planned for next week. It would be great, if you can split this PR more into smaller ones, so that it is easy for us to review and merge. thanks!

**Comment by dvesh3** [comment]
@blankse please let us know when you are done with splitting? thanks!

**Comment by blankse** [comment]
@dvesh3 I can split more on weekend but it would help when you can review/merge the existing one. So I have fewer to look through.

**Comment by dvesh3** [comment]
@blankse the existing PRs are merged, so you can rebase and continue with splitting. thanks a lot 😺

**Comment by blankse** [comment]
@dvesh3 I'm sorry, I didn't find time for it this weekend. I will splitting PRs today.

**Comment by dvesh3** [comment]
> @dvesh3 I'm sorry, I didn't find time for it this weekend. I will splitting PRs today.

Don't be!! we anyway don't expect you to be fiddling with code on weekends 😉

**Comment by dvesh3** [comment]
@blankse please resolve the conflicts. Should we start reviewing this PR or there are more split-ups coming?

**Comment by blankse** [comment]
@dvesh3 I can still do some split PRs today

**Comment by blankse** [comment]
@dvesh3 Should I create a PR with only bump PhpStan Level to 5 and a bigger baseline file? So we can fix some small errors in 1.4.1 and avoid new errors?

File: lib/Google/Cse.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ public function readGoogleResponse(Search $googleResponse)
     {
         $items = [];
 
-        $this->setRaw($googleResponse);
+        $this->setRaw((array) $googleResponse);
```
Comment (dvesh3) [inline lib/Google/Cse.php]:
```suggestion
        $this->setRaw($googleResponse->getItems());
```

File: lib/Messenger/Handler/GenerateWeb2PrintPdfHandler.php [hunk 2]
```diff
@@ -31,7 +31,7 @@ public function __construct(protected LoggerInterface $logger)
 
     public function __invoke(GenerateWeb2PrintPdfMessage $message)
     {
-        $generationId = $message->getProcessId();
+        $documentId = (int) $message->getProcessId();
```
Comment (dvesh3) [inline lib/Messenger/Handler/GenerateWeb2PrintPdfHandler.php]: we change `GenerateWeb2PrintPdfMessage::$processId` type to int

File: lib/Web2Print/Processor.php [hunk 3]
```diff
@@ -82,7 +82,7 @@ public function preparePdfGeneration($documentId, $config)
 
         if (!$disableBackgroundExecution) {
             \Pimcore::getContainer()->get('messenger.bus.pimcore-core')->dispatch(
-                new GenerateWeb2PrintPdfMessage($jobConfig->documentId)
+                new GenerateWeb2PrintPdfMessage((string) $jobConfig->documentId)
```
Comment (dvesh3) [inline lib/Web2Print/Processor.php]: we change GenerateWeb2PrintPdfMessage::$processId type to int

File: models/DataObject/ClassDefinition/Data/Objectbricks.php [hunk 4]
```diff
@@ -183,7 +183,7 @@ private function doGetDataForEditmode($getter, $data, $params, $allowedBrickType
 
         foreach ($calculatedChilds as $fd) {
             $fieldData = new DataObject\Data\CalculatedValue($fd->getName());
-            $fieldData->setContextualData('objectbrick', $this->getName(), $allowedBrickType, $fd->getName(), null, null, $fd);
+            $fieldData->setContextualData('objectbrick', $this->getName(), null, $fd->getName(), null, null, $fd);
```
Comment (dvesh3) [inline models/DataObject/ClassDefinition/Data/Objectbricks.php]:
```suggestion
            $fieldData->setContextualData('objectbrick', $this->getName(), null, null, null, $fd->getName(), $fd);
```

File: lib/Messenger/Handler/GenerateWeb2PrintPdfHandler.php (hunk 2 above)
Comment (blankse) [reply lib/Messenger/Handler/GenerateWeb2PrintPdfHandler.php]: Changed it https://github.com/pimcore/pimcore/pull/11826/commits/18c25300ce9e8808a23c40a68503498c1f102788

File: lib/Web2Print/Processor.php (hunk 3 above)
Comment (blankse) [reply lib/Web2Print/Processor.php]: Changed it https://github.com/pimcore/pimcore/pull/11826/commits/18c25300ce9e8808a23c40a68503498c1f102788

**Review by dvesh3 — APPROVED** [review]

**Comment by dvesh3** [comment]
@blankse nice work 👏 thanks a lot!

**Pull request merged**